src/content/docs/three-js/three-js-case-study.md

```diff
@@ -0,0 +1,148 @@
+
+While only having a small community, Babylon.js, is one of the main competitors to Three.js and offers support for Apple's AR goggles. Babylon.js presents a more professional appeal and delivers visual fidelity that surpasses both Three.js and Whitestorm [slant.co] [babylonjs.com].
+
+PlayCanvas, a third competitor, claims to have excellent cross-platform compatibility. It boasts similar features as both Three.js and Babylon.js. It provides users with many tools geared towards game development, although some of its features are hidden behind a paid subscription [slant.co] [playcanvas.com].
+
+One of Three.js’ main strengths appears to be the great documentation provided by the developers and also its large number of official plugins and tutorials [slant.co].
+
+###The following table compares Three.js with its main competitors across various key points:
+
+| Three.js     | Babylon.js | Whitestorm | PlayCanvas |
+| ------------ | ---------- | ---------- | ---------- |
+| Physics-     | Physics-   | Physics-   | Physics-   |
+| engine:No\*  | engine:Yes | engine:Yes | engine:Yes |
+|              |            |            |            |
+| Community:   | Community: | Community: | Community: |
+| Large        | Small      | Small      | Large      |
+|              |            |            |            |
+|              |
+| Features:    | Features:  | Features:  | Features:  |
+| Rich         | Rich       | Medium     | Rich\*\*\* |
+|              |            |            |            |
+|              |
+| Fidelity:    | Fidelity:  | Fidelity   | Fidelity   |
+| Mid/high\*\* | High       | low        | High       |
+|              |            |            |            |
+| License: N   | License: N | License: N | License: Y |
+
+Y = Yes, N = No
+\*Three.js does not include its own physics engine, but is fully compatible with external physics frameworks for JavaScript, as explained in this article: "https://en.threejs-university.com/2021/08/24/using-physics-in-three-js-with-ammo-jsintroduction/" [threejs-university.com].
+
+\*\* When comparing Three with Babylon, the level of fidelity seems a little lower.
+
+\*\*\* PlayCanvas offers great cross-platform compatibility[slant.co].
+
+NB: PlayCanvas Offers a free license, as long as the project-size is kept below a certain size [playcanvas.com].
+
+### Market Comparison Conclusion
+
+Each of the aforementioned frameworks has its own strengths and weaknesses when compared to each other. The choice of which framework to use for displaying three-dimensional content to an audience is ultimately up to you. However, Three.js is a strong contender due to its large community, free-to-use features, and official plugins.
+
+## Getting Started
+
+## Getting started and setting up the project.
```

Comment:
Seems like a double heading?

Reply:
I have now removed the first heading. I have also rewritten the links with proper Markdown syntax. I will will push my changes soon!